src/main/res/local_config_winsdk.bzl

```diff
@@ -0,0 +1,66 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//tools/cpp:windows_cc_configure.bzl", "find_vc_path", "setup_vc_env_vars")
+load("//tools/cpp:cc_configure.bzl", "MSVC_ENVVARS")
+
+def _find_rc_exe(repository_ctx):
+    vc = find_vc_path(repository_ctx)
+    if vc:
+        env = setup_vc_env_vars(
+            repository_ctx,
+            vc,
+            envvars = [
+                "WindowsSdkDir",
+                "WindowsSdkVerBinPath",
+            ],
+            allow_empty = True,
+            escape = False,
+        )
+
+        # Try the versioned directory.
+        sdk = env.get("WindowsSdkVerBinPath")
+        if sdk:
+            exe = repository_ctx.path(sdk).get_child("x64").get_child("rc.exe")
```

Comment:
Can we add an attribute to allow users to specify the target platform?
You can find `arm`, `arm64`, `x64`, `x86` under `C:\Program Files (x86)\Windows Kits\10\bin\10.0.15063.0`

I'm planning to add this toolchain support for all those target platforms, it would be nice if the resource compiling also support them.

Reply:
Good idea.
That way we could create separate toolchains, each with their own `exec_compatible_with`.
Does that match your idea?

Reply:
Oh, that sounds good! But do we also need to add `target_compatible_with`?

Reply:
You're right, that's what we actually need.
I'll update the PR and ping the thread.